mix.exs

```diff
@@ -18,7 +24,7 @@ defmodule Binance.MixProject do
     [
       mod: {Binance.Supervisor, []},
       applications: [:exconstructor, :poison, :httpoison],
-      extra_applications: [:logger]
+      extra_applications: [:logger, :exvcr]
```

Comment:
What the reason for adding the `exvcr` to `extra_applications`?

Reply:
Ah, thanks for reminding this to me...
I first made this change to avoid this warning when running `mix test` in binance.ex : 
```
You have configured application :exvcr in your configuration file,
but the application is not available.

This usually means one of:

  1. You have not added the application as a dependency in a mix.exs file.

  2. You are configuring an application that does not really exist.

Please ensure :exvcr exists or remove the configuration.

```
In case it helps reproducing/explaining, my elixir configuration is:
```
Erlang/OTP 23 [erts-11.1.3] [source] [64-bit] [smp:12:12] [ds:12:12:10] [async-threads:1] [hipe]

IEx 1.11.2 (compiled with Erlang/OTP 23)
```

However this change somehow triggered another warning in a client code where I use binance.ex (something about :exvcr application not being defined, I can't recall exactly or reproduce right now... I think it was a case of :exvcr being used only for tests, but not in other mix environments...)

So, anyway, I removed that `:exvcr` app dependency from the branch of my fork that I am currently using, and kept the warning for now.
I should also remove it from this PR. I'll make a commit for this shortly...

Maybe there is a proper way to fix that warning ? (lets do it in another issue).

Reply:
For context I had a similar issue for bypass, which I fixed like this : https://github.com/asmodehn/binance.ex/pull/1/commits/3b737793150d6977c444ceefa3d845a611896c98
But it doesn't seem to work for `:exvcr` for some reason... I didn't dig further just yet, it just a minor annoyance.

Reply:
Ahhh this issue - I forgot about it - I was honestly just ignoring it whenever I was updating this package.

I'm not sure about the proper way of fixing this but adding it to the list of applications would cause it to be started in the "production" which we don't want as well as the dependencies list doesn't allow (it's in "test only" mode).

I would focus on this issue in a separate PR/issue but either way, it's not a deal-breaker IMO, users of the package will never see it (but it could be indeed confusing to the contributors so I appreciate your eagerness to fix it!)